lib/diplomat/key_utils.ex

```diff
@@ -0,0 +1,200 @@
+defmodule KeyUtils do
```

Comment:
Can you help me understand what this is for? It doesn't seem to be used in any of the code (unless I'm missing it).